lib/beacon/dapp_client.mli

```diff
@@ -88,6 +88,14 @@ val get_accounts : t -> Account_info.t list Lwt.t
     account. *)
 val get_active_account : t -> Account_info.t option Lwt.t
 
+(** [request_broadcast ?network ~signed_transaction client] request the
+    injection of an already-signed transaction to the network. *)
+val request_broadcast
+  :  ?network:Network.t
+  -> signed_transaction:string
+  -> t
+  -> Transaction_hash_response_output.t Lwt.t
```

Comment:
Pretty sure that the promise can be rejected so I suggest to wrap the `transaction_hash_response_output` into a `result`.